### ISSUE TITLE
Power on USB ports?

### ISSUE THREAD

**Issue by Juno63** [Xinyuan-LilyGO/LilyPi#2]
There seems to be no power (3.5V/5V) on the USB ports. Is there a special setting to activate the power on USB ports?

**Comment by Juno63** [comment]
(GND an GPIO pins seem to work well).

**Comment by Juno63** [comment]
LilyPi ILI9481 (GT911) version.

**Comment by LilyGO** [comment]
You need to switch between 3.3 and 5V

**Comment by Juno63** [comment]
No, that is not the solution. There seems to be no 3.5V and no 5V either, LED between GPIO-pins and GND works, LED between USB POWER (3.5V/5V) and GND does not work.

**Comment by LilyGO** [comment]
Can you take a picture and see what you're measuring?

**Comment by Juno63** [comment]
![LilyPi_blink](https://user-images.githubusercontent.com/101507582/159468436-401e29bd-7959-4d90-978a-c17aab6ffd28.jpg)
Using the basic Blink sketch on pin 25 (or 26) works fine. The red LED is blinking like it should. But the green LED, which is between GND and + (3.5V/5V) isn't blinking at all.
This is the same when measuring with my multimeter: Pin 25 (high) to GND: 3.5V (or 5V). POWER to GND: 0 V.

**Comment by LilyGO** [comment]
USB power switch in 5V
![1](https://user-images.githubusercontent.com/31160602/159605967-27c22fbb-6f83-4626-930b-c451a8909d73.png)
?

**Comment by Juno63** [comment]
I tried. Doesn't work. There is no power at all on POWER (3V3/5V).

**Comment by LilyGO** [comment]
How about using Type-C power?

**Comment by Juno63** [comment]
That is what I am trying to do with this USB 3.1 Type-C Screw Terminal Block Connector. But there is no power at all on the POWER pin (3V3/5V).

**Comment by LilyGO** [comment]
![1](https://user-images.githubusercontent.com/31160602/160047807-3f92c0fd-e3a0-4be6-8f70-623e8e1f77ba.png)

**Comment by Juno63** [comment]
How to connect i2c sensors to USB port if there is no power on USB power pin?

**Comment by jaccoT** [comment]
in setup just ttgo->turnOnUsb();

this activates the USB ports.

**Comment by jwktje** [comment]
What is the maximum amperage we can get from the USB ports?
